ui/imports/shared/views/HistoryView.qml

```diff
@@ -99,6 +102,8 @@ ColumnLayout {
 
         delegate: transactionDelegate
 
+        header: headerComp
```

Comment:
May be more general header issue, but when you scroll history the button is visible under the delegates and sections
![image](https://github.com/status-im/status-desktop/assets/11396062/04680495-58bf-4e80-8569-85acb80dd6f4)